File: CoronaManagement/Client/src/models/VaccinationDetail.ts
import { Users } from './Users'; 
import { VaccineManufacturer } from './VaccineManufacturer'; 

export class VaccinationDetail {
  constructor(
    public vaccinationId: number,
    public userId: number | null,
    public vaccineDate: Date | null,
    public vaccineManufacturerId: number | null,
    public user: Users | null,
    public vaccineManufacturer: VaccineManufacturer | null
  ) {}
}

File: CoronaManagement/Server/Entity/Models/VaccinationDetail.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace Entity
{
    public partial class VaccinationDetailTbl
    {
        [Key]
        public int VaccinationId { get; set; }
        public int? UserId { get; set; }
        public DateTime? VaccineDate { get; set; }
        public int? VaccineManufacturerId { get; set; }

        public virtual UserTbl? User { get; set; }
        public virtual VaccineManufacturerTbl? VaccineManufacturer { get; set; }
    }
}
